CHANGELOG.md

```diff
@@ -4921,10 +4924,21 @@ patch operations can re-use these classes for generating patch messages.
 
 ## [0.14.1]
 
+<<<<<<< HEAD
 [Unreleased]: https://github.com/linkedin/rest.li/compare/v29.18.2...master
 [29.18.2]: https://github.com/linkedin/rest.li/compare/v29.18.1...v29.18.2
+=======
+<<<<<<< HEAD
+[Unreleased]: https://github.com/linkedin/rest.li/compare/v29.18.1...master
+>>>>>>> 5cd01279a (Use runtimeClasspath instead of runtime config.)
 [29.18.1]: https://github.com/linkedin/rest.li/compare/v29.18.0...v29.18.1
 [29.18.0]: https://github.com/linkedin/rest.li/compare/v29.17.4...v29.18.0
+=======
+[Unreleased]: https://github.com/linkedin/rest.li/compare/v29.18.3...master
+[29.18.3]: https://github.com/linkedin/rest.li/compare/v29.18.2...v29.18.3
+[29.18.2]: https://github.com/linkedin/rest.li/compare/v29.17.5...v29.18.2
+[29.17.5]: https://github.com/linkedin/rest.li/compare/v29.17.4...v29.17.5
+>>>>>>> e4749f625 (Use runtimeClasspath instead of runtime config.)
```

Comment:
Looks like there are some conflicts which need to be fixed.

Reply:
Good catch! Fixed.